src/modelbench/run.py

```diff
@@ -243,21 +266,23 @@ def update_standards_to(standards_file):
     reference_suts = [
         ModelGaugeSut.for_key("mistral-7b"),
     ]
-    all_benchmarks = [
-        GeneralPurposeAiChatBenchmark(),
-        GeneralPurposeAiChatBenchmarkV1(Locale.EN_US),
-        GeneralPurposeAiChatBenchmarkV1(Locale.FR_FR),
-        GeneralPurposeAiChatBenchmarkV1(Locale.HI_IN),
-        GeneralPurposeAiChatBenchmarkV1(Locale.ZH_CN),
-    ]
-    run_result = run_benchmarks_for_suts(all_benchmarks, reference_suts, None)
-    hazards = set(itertools.chain.from_iterable([b.hazards() for b in run_result.benchmarks]))
-    all_hazard_numeric_scores = {h.uid: [] for h in hazards}
+    if not ensure_ensemble_annotators_loaded():
+        print("Can't load private annotators needed for calibration")
+        exit(1)
+
+    benchmarks = [GeneralPurposeAiChatBenchmark()]
+    for l in Locale:
+        benchmarks.append(GeneralPurposeAiChatBenchmarkV1(l, "ensemble"))
+    run_result = run_benchmarks_for_suts(benchmarks, reference_suts, 100)
+    all_hazard_numeric_scores = defaultdict(list)
     for benchmark, scores_by_sut in run_result.benchmark_scores.items():
+        print(benchmark)
```

Comment:
Needed?

Reply:
Totally not. Debugging code I left in. Thanks for catching it.